src/fights.ts

```diff
@@ -578,20 +579,18 @@ class FreeRunFight extends FreeFight {
     if (!this.available()) return;
     if ((this.options.cost ? this.options.cost() : 0) > get("garbo_valueOfFreeFight", 2000)) return;
     while (this.available()) {
-      const runSource = findRun(this.options.familiar ? false : true);
+      const runSource = tryFindFreeRun(this.options.familiar ? { noFamiliar: () => true } : {});
```

Comment:
is there a reason it's not `{ noFamiliar: () => this.options.familiar !== undefined }`?

Reply:
sure